spec/checkPullRequestReviewSpec.js

```diff
@@ -1,11 +1,8 @@
 // Copyright 2020 The Oppia Authors. All Rights Reserved.
-//
 // Licensed under the Apache License, Version 2.0 (the "License");
 // you may not use this file except in compliance with the License.
 // You may obtain a copy of the License at
-//
```

Comment:
Unneeded change.

Reply:
Correcting it in the latest commit Sir.Thanks.

Reply:
Correcting it in the latest commit Sir.Thanks.

Reply:
No, it wasn't fixed.

Reply:
Fixed them in the later commits. Sorry for Later reply.

Reply:
@vojtechjelinek  Sorry for the previous one Sir. Could you please take a look now and tell me if it is alright now. Thanks.